Cargo.toml

```diff
@@ -26,7 +26,7 @@ bytes = "1.0"
 cadence = "0.26"
 chrono = "0.4"
 docopt = "1.1"
-cloud-storage = "0.6" # 0.7+ includes request 0.11, tokio 1.4
+cloud-storage = { git = "https://github.com/mozilla-services/cloud-storage-rs", branch = "feat/create_with_params-0.6.2" } # 0.7+ includes request 0.11, tokio 1.4
```

Comment:
this branch could use a separate review, I can set up a PR to merge it to another branch for that sake